web/src/integrationTest/kotlin/no/kartverket/matrikkel/bygning/v1/common/JWTUtil.kt

```diff
@@ -0,0 +1,30 @@
+package no.kartverket.matrikkel.bygning.v1.common
+
+import no.nav.security.mock.oauth2.MockOAuth2Server
+
+class JWTUtil() {
```

Comment:
Bare for å tydeliggjøre at den jobber på MockOAuth2Server klassen:

```suggestion
class MockOAuth2ServerExtensions() {
```